Dockerfile

```diff
@@ -1,6 +1,8 @@
-FROM python:3.6.8-stretch
+FROM python:3.6.8-slim-stretch as builder
```

Comment:
Do we really need to build with `3.6.8`? How do you think to use `3.6.5 ` here too?